go-controller/pkg/node/controllers/upgrade/node_upgrade.go

```diff
@@ -90,13 +109,15 @@ func (uc *upgradeController) onNodeUpdate(oldObj, newObj interface{}) {
 	}
 }
 
-func (uc *upgradeController) Run(stopCh <-chan struct{}, informerStop chan struct{}) error {
+func (uc *upgradeController) Run(stopCh <-chan struct{}) error {
 	defer utilruntime.HandleCrash()
 	defer uc.queue.ShutDown()
 
 	klog.Infof("Starting controller %s", controllerName)
 	defer klog.Infof("Shutting down controller %s", controllerName)
 
+	informerStop := make(chan struct{})
```

Comment:
@girishmg what closes this stop channel?

Reply:
@dcbw the current code doesn't close it as well. Will address as a follow on PR, is that ok?